python/ray/_private/services.py

```diff
@@ -1501,10 +1502,19 @@ def start_gcs_server(
         command += [f"--redis_username={redis_username}"]
     if redis_password:
         command += [f"--redis_password={redis_password}"]
+
+    # Logging is fully managed by C++ side spdlog, which supports rotation and file
+    # count limitation.
+    stderr_file = (
+        sys.stderr
+        if os.environ.get("RAY_LOG_TO_STDERR", "0") == "1"
+        else open(os.devnull, "w")
+    )
     process_info = start_ray_process(
         command,
         ray_constants.PROCESS_TYPE_GCS_SERVER,
-        stdout_file=stdout_file,
+        # GCS server stdout is completely taken over by C++ side spdlog, or disabled.
+        stdout_file=open(os.devnull, "w"),
         stderr_file=stderr_file,
```

Comment:
I think now both should always be `None`. We no longer need to redirect stdout and stderr.

Reply:
I think last time it was you who prompts me we have certain env var?
https://github.com/ray-project/ray/blob/c116fec44f89da2edbd00bcb6269d200f2727ecd/python/ray/_private/services.py#L1508-L1512

Reply:
Updated.